topbeat/docs/fields.asciidoc

```diff
@@ -168,12 +167,12 @@ type: int
 
 The amount of CPU time spent in involuntary wait by the virtual CPU while the hypervisor was servicing another processor. Available only on Unix.
 
-[float]
+
 === cpus Fields
 
 This group contains CPU usage per core statistics.
 
-[float]
```

Comment:
@monicasarbu @dedemorton [float] was removed when I was running `make update`. Was this introduced manually? If yes, is it still required?